runtime/compilers/rillv1/parse_api.go

```diff
@@ -88,20 +98,23 @@ func (p *Parser) parseAPI(node *Node) error {
 	r.APISpec.OpenapiSummary = openapiSummary
 	r.APISpec.OpenapiParameters = openapiParams
 	r.APISpec.OpenapiResponseSchema = openapiSchema
+	r.APISpec.SecurityRules = securityRules
 
 	return nil
 }
 
 // DataYAML is the raw YAML structure of a sub-property for defining a data resolver and properties.
 // It is used across multiple resources, usually under "data:", but inlined for APIs.
 type DataYAML struct {
-	Connector      string         `yaml:"connector"`
-	SQL            string         `yaml:"sql"`
-	MetricsSQL     string         `yaml:"metrics_sql"`
-	API            string         `yaml:"api"`
-	Args           map[string]any `yaml:"args"`
-	Glob           yaml.Node      `yaml:"glob"` // Path (string) or properties (map[string]any)
-	ResourceStatus map[string]any `yaml:"resource_status"`
+	Connector            string             `yaml:"connector"`
+	SQL                  string             `yaml:"sql"`
+	MetricsSQL           string             `yaml:"metrics_sql"`
+	API                  string             `yaml:"api"`
+	Args                 map[string]any     `yaml:"args"`
+	Glob                 yaml.Node          `yaml:"glob"` // Path (string) or properties (map[string]any)
+	ResourceStatus       map[string]any     `yaml:"resource_status"`
+	Security             SecurityPolicyYAML `yaml:"security"`
```

Comment:
I also think this should be a pointer, i.e. `*SecurityPolicyYAML` (not sure if it matters, but we use that the other places and the YAML library has subtle differences in its treatment of pointer vs. literal types, so probably best to stay consistent with the other places its used)